onnx/backend/test/loader/__init__.py

```diff
@@ -8,13 +8,9 @@
 
 from onnx.backend.test.case.test_case import TestCase
 
-DATA_DIR = os.path.join(
-    os.path.dirname(os.path.realpath(os.path.dirname(__file__))), "data"
-)
-
 
 def load_model_tests(
-    data_dir: str = DATA_DIR,
+    data_dir: str,
```

Comment:
```suggestion
    data_dir: str | os.PathLike,
```

we can make it take a pathlike object